src/app/teams/teams-member.component.html

```
@@ -1,6 +1,6 @@
 <mat-card-header>
-  <img mat-card-avatar [src]="member?.avatar" class="cursor-pointer avatar-spacing" (click)="openMemberDialog(member)">
-  <a mat-card-title class="cursor-pointer member-name" (click)="openMemberDialog(member)">{{member?.userDoc?.fullName || member?.userDoc?.firstName || member?.name | truncateText:70}}</a>
+  <img mat-card-avatar [src]="member?.avatar" tabindex="0" role="button" class="cursor-pointer avatar-spacing" (keydown.enter)="openMemberDialog(member, $event)" (keydown.space)="openMemberDialog(member, $event)" (click)="openMemberDialog(member, $event)">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)